frontend/models/feedback-task-model/feedback-dataset/FeedbackDataset.model.js

```diff
@@ -0,0 +1,25 @@
+import { Model } from "@vuex-orm/core";
+import { GlobalQuestion } from "../global-question/GlobalQuestion.model";
+import { Record } from "../record/Record.model";
+
+class FeedbackDataset extends Model {
+  static entity = "feedbackDatasets";
+
+  static fields() {
+    return {
+      id: this.uid(),
+      name: this.string(""),
+      guidelines: this.string(""),
+      workspace_id: this.attr(null),
+      workspace_name: this.attr(null),
+      inserted_at: this.attr(null),
+      updated_at: this.attr(null),
+
+      // relationships
+      global_questions: this.hasMany(GlobalQuestion, "dataset_id"),
```

Comment:
Regarding the naming, I think this could be renamed to `questions_settings`, and the entity to `QuestionSettings`. Please, correct me if I'm wrong.

Reply:
Or maybe just, `questions` and `Question`. Do you have a `LocalQuestion` equivalent model?

Reply:
- In the point of view of the front app, I think it's a little confusing to use "settings" here, since we have differents component using this prefix too. 
- The "global" here refer to a model attach to the dataset and not in nested entity (like a global variable) but maybe there are better naming. if I put only "question" as the name of the entity, I feel a little confusing if an entity called "Question" have also attribut called "question"
=> I try to use as possible the JS convention to have two word in variables, so a compromise could be "dataset_questions" ?

Reply:
> * The "global" here refer to a model attach to the dataset and not in nested entity (like a global variable) but maybe there are better naming. if I put only "question" as the name of the entity, I feel a little confusing if an entity called "Question" have also attribut called "question"
> 
For our model, we have an attribute called `title`, instead of `question`.  It's more general and can fit better if we have feedback components without an explicit question (for example a text area for `Notes`)
```json
{
    "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
    "name": "string",
    "title": "string",
    "required": true,
    "settings": {
      "type": "text"
    },
    "inserted_at": "2023-04-14T12:16:21.823Z",
    "updated_at": "2023-04-14T12:16:21.823Z"
  }
```
> => I try to use as possible the JS convention to have two word in naming, so a compromise could be "dataset_questions" instead of "global_questions" ?

Sounds good to me.